satpy/writers/cf_writer.py

```diff
@@ -785,17 +796,20 @@ def _remove_satpy_attributes(new_data):
     @staticmethod
     def update_encoding(dataset, to_netcdf_kwargs):
         """Update encoding info (deprecated)."""
-        warnings.warn('CFWriter.update_encoding is deprecated. '
-                      'Use satpy.writers.cf_writer.update_encoding instead.',
-                      DeprecationWarning)
+        warnings.warn(
+            'CFWriter.update_encoding is deprecated. '
+            'Use satpy.writers.cf_writer.update_encoding instead.',
+            DeprecationWarning,
+            stacklevel=2
+        )
         return update_encoding(dataset, to_netcdf_kwargs)
```

Comment:
How long has this been deprecated and why is it even a public method? Can we just remove this?

Reply:
This has been deprecated for 3 years now, I think we can remove it. Probably shouldn't have been public in the first place.